astropy/io/votable/tree.py

```diff
@@ -1819,6 +1820,7 @@ def __init__(
         equinox=None,
         epoch=None,
         system=None,
+        refposition=None,
```

Comment:
this is technically a backwards incompatible API change as these could be used as positional arguments.

So: maybe it's time to make them mandatory kwargs?

Reply:
I moved the new arg to the end of the list so that it wouldn't break backward compatibility.  Mandatory kwargs seems like a good improvement for all these elements as a separate change.